### ISSUE TITLE
Some species aren't processed well in py3Dmol

### ISSUE THREAD

**Issue by alongd** [ReactionMechanismGenerator/ARC#126]
Could also be related to atom mapping to / from RDKit

```
O      -0.98526500    1.74795900    0.84621100
C      -1.15584400    0.77365100   -0.08180100
H      -2.21814200    0.49837200   -0.21097000
H      -0.86607200    1.22917700   -1.05225000
C      -0.28520800   -0.49049600    0.12747900
C       1.20033300   -0.14651100    0.18038000
H       1.79449700   -1.04204200    0.37859600
H       1.52877700    0.29545700   -0.76226600
H       1.38216500    0.57981500    0.97402900
C      -0.74563500   -1.24665600    1.37315400
H      -0.63589300   -0.59990000    2.24646500
H      -1.79478200   -1.53611900    1.28112200
H      -0.14716900   -2.14752600    1.52233500
O      -0.58693200   -1.42025300   -0.93733200
O      -0.25339300   -0.81210500   -2.21637000
H       0.52665200   -1.32818200   -2.46076100
```

![Screenshot_2019-05-01 plot xyz](https://user-images.githubusercontent.com/16158262/57034573-e4346080-6c1d-11e9-8760-c42bbed84d91.png)

**Comment by alongd** [comment]
methyl_naphthalene_rad optimized at wb97xd/6-311++g(d,p)
C       1.67292700   -1.06839200   -1.09699000
C       2.91128900   -1.28615800   -1.75539400
C       0.72911800   -0.10301100   -1.65560800
C       1.10740800    0.57229500   -2.84074600
C       3.22200900   -0.56832800   -2.94526500
C       3.82796900   -2.22000500   -1.21785700
C       1.40590500   -1.80001400    0.07775300
C       2.32886700    0.34080100   -3.46801300
C       3.53867300   -2.91673500   -0.07235900
C       2.31322900   -2.70285000    0.58130900
C      -0.50377000    0.17470100   -1.08091900
H       0.41486400    1.29147700   -3.26497900
H       2.57482700    0.88240600   -4.37458700
H       4.17400200   -0.75019500   -3.43223700
H       4.77104400   -2.37657800   -1.73155500
H       4.25021800   -3.62948900    0.32881400
H       2.08129400   -3.25184000    1.48695400
H       0.46961600   -1.65531000    0.60200400
H      -0.85749600   -0.30231700   -0.17760100
H      -1.16148800    0.90162600   -1.54205900

![image](https://user-images.githubusercontent.com/16158262/57454883-ddb67200-7237-11e9-9bd6-43f597a330cc.png)


![image](https://user-images.githubusercontent.com/16158262/57454894-e6a74380-7237-11e9-86d7-9807436dc3e3.png)

**Comment by alongd** [comment]
This was resolved, probably stem from a [bug ](61cb320be1a80856130358f74f173a485272688f)of displaying the molecules (which changes atom order) before attempting to plot them in 3D. Now the tool works great:
![12](https://user-images.githubusercontent.com/16158262/71774423-71158380-2f3c-11ea-9e88-bfca3c5ccefd.png)
![11](https://user-images.githubusercontent.com/16158262/71774424-72df4700-2f3c-11ea-9bb4-4d13bcdefa6e.png)
